experanto/interpolators.py

```diff
@@ -419,11 +424,26 @@ def _parse_trials(self) -> None:
         metadatas, keys = self.read_combined_meta()
 
         for key, metadata in zip(keys, metadatas):
-            data_file_name = self.root_folder / "data" / f"{key}.npy"
+            format = metadata.get("file_format")
+
+            if format is None:
+                format = ".npy"
+
+            if self.image_names:
+                image_name = metadata.get("image_name")
+                data_file_name = (
+                    self.root_folder.parent / "stimuli" / f"{image_name}{format}"
+                )
+            else:
+                data_file_name = self.root_folder / "data" / f"{key}{format}"
```

Comment:
I guess this should also be fixed (e.g. there should be no `stimuli` and even in case of `image_names = True` the files should live in the `f'{self.root_folder}/data/'`